app/src/main/java/agency/five/codebase/android/movieapp/ui/component/UserScoreProgressBar.kt

```diff
@@ -0,0 +1,62 @@
+
+
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun UserScoreProgressBar(
+    userScore: Float,
+    modifier: Modifier = Modifier
+){
+    Box(modifier = modifier){
+        Canvas(modifier = modifier
+            .size(90.dp)
+            .padding(10.dp)){
+            drawArc(
+                color = Color.Green,
+                alpha = 0.2f,
+                startAngle = 0f,
+                sweepAngle = 360f,
+                useCenter = false,
+                style = Stroke(
+                    width = 24f,
+                    cap = StrokeCap.Round
+                )
+            )
+            drawArc(
+                color = Color.Green,
+                startAngle = 270f,
+                sweepAngle = userScore*360f,
+                useCenter = false,
+                style = Stroke(
+                    width = 24f,
+                    cap = StrokeCap.Round
+                )
+            )
```

Comment:
Here you have a lot of [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)), please extract them to constants on top of the file. Also, make sure to reformat code so it automatically adds whitespace between numbers and '*' characters.